charts/embedded-cluster-operator/templates/embedded-cluster-logs-collector.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: embedded-cluster-cluster-support-bundle
+  labels:
+    troubleshoot.sh/kind: support-bundle
+    {{- with (include "embedded-cluster-operator.labels" $ | fromYaml) }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+data:
+  support-bundle-spec: |
+    apiVersion: troubleshoot.sh/v1beta2
+    kind: SupportBundle
+    metadata:
+      name: embedded-cluster-logs-collector
+      labels:
+        troubleshoot.sh/kind: support-bundle
+    spec:
+      collectors:
+        - runDaemonSet:
+            name: "k0scontroller"
```

Comment:
I think we also need to have another collector for the k0sworker systemd unit.

Reply:
added